latis-python-client/tests/example.py

```diff
@@ -0,0 +1,64 @@
+import os
+import sys
+import platform
+
+libPath = os.path.dirname(os.path.realpath(__file__))
+if platform.system() == 'Windows':
+    libPath = libPath.replace('\\latis-python-client\\tests',
+                              '\\latis-python-client\\client')
+else:
+    libPath = libPath.replace('/latis-python-client/tests',
+                              '/latis-python-client/client')
+
+sys.path.insert(0, libPath)
+
+import latis
+
+print('Creating Latis Instance\n')
+# 1 - Create Instance
+instance = latis.LatisInstance(
+    baseUrl='https://lasp.colorado.edu/lisird/latis',
+    latis3=False)
+
+instance3 = latis.LatisInstance(
+    baseUrl='https://lasp.colorado.edu/lisird/latis',
+    latis3=True)
+
+print('\nSearch Catalog\n')
+# 2 - Search Catalog
+# print(instance.catalog.list) # (Full catalog)
+print(instance.catalog.search('cls'))
+print(instance3.catalog.search('sorce'))
+print(instance.catalog.catalog)
+print(instance3.catalog.catalog)
+
+print('\nCreating Datasets\n')
+# 3 - Create a dataset objects
+clsRadioFluxF8 = instance.createDataset('cls_radio_flux_f8')
+clsRadioFluxF15 = instance.createDataset('cls_radio_flux_f15')
+sorceMGIndex = instance3.createDataset('sorce_mg_index')
+
+print('\nCreating Queries\n')
+# 4 - Create queries
+queryReturn = clsRadioFluxF8.buildQuery()
```

Comment:
Don't require running this. Can the query building happen behind the scenes? Maybe keep a list of operations as the state instead of the query itself?